README.rst

```diff
@@ -260,3 +260,10 @@ To get statement coverage::
 
     $ coverage run --source edx_sga manage.py lms --settings=test test edx_sga
     $ coverage report -m
+
+To run JavaScript tests, install js-test-tool_ and run::
+
+    $ js-test-tool run ./edx_sga/static/js_test.yml --use-firefox
```

Comment:
The tests can be run outside of the devstack currently so the relative path should work fine. I'll make an issue to get the test runner to work together with the test runner in the devstack.